pvanalytics/quality/util.py

```diff
@@ -58,3 +58,37 @@ def check_limits(val, lower_bound=None, upper_bound=None,
         return ub_op(val, upper_bound)
     else:
         raise ValueError('must provide either upper or lower bound')
+
+
+def daily_maxmin(series, maximum, inclusive=False):
```

Comment:
```suggestion
def daily_min(series, daily_min, inclusive=False):
```
Still don't like the name